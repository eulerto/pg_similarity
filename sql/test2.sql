--
-- errors
--
SET pg_similarity.cosine_threshold to 1.1;
SET pg_similarity.qgram_tokenizer to 'alnum';

--
-- valid values
--
SET pg_similarity.block_is_normalized to true;
SET pg_similarity.cosine_threshold = 0.72;
SET pg_similarity.dice_tokenizer to 'alnum';
SET pg_similarity.euclidean_is_normalized to false;
SET pg_similarity.jarowinkler_is_normalized to false;
SET pg_similarity.qgram_tokenizer to 'gram';
