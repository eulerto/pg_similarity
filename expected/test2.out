--
-- errors
--
SET pg_similarity.cosine_threshold to 1.1;
ERROR:  1.1 is outside the valid range for parameter "pg_similarity.cosine_threshold" (0 .. 1)
SET pg_similarity.qgram_tokenizer to 'alnum';
ERROR:  invalid value for parameter "pg_similarity.qgram_tokenizer": "alnum"
HINT:  Available values: gram.
--
-- valid values
--
SET pg_similarity.block_is_normalized to true;
SET pg_similarity.cosine_threshold = 0.72;
SET pg_similarity.dice_tokenizer to 'alnum';
SET pg_similarity.euclidean_is_normalized to false;
SET pg_similarity.jarowinkler_is_normalized to false;
SET pg_similarity.qgram_tokenizer to 'gram';
